Makefile

```diff
@@ -155,6 +155,10 @@ installer: ## Builds the container image for the installer and outputs it to the
 talos: ## Builds the Talos container image and outputs it to the artifact directory.
 	@$(MAKE) registry-$@ TARGET_ARGS="--allow security.insecure"
 
+.PHONY: talosctl-image
+talosctl-image: ## Builds the talosctl container image and outputs it to the artifact directory.
+	@$(MAKE) registry-talosctl TARGET_ARGS="--allow security.insecure"
```

Comment:
a bit of a question, but we have talosctl built for more arches than Talos itself

the way it is right now, it should create talosctl image for amd64/arm64, but we certainly could publish more, and that requires some bigger changes, so I would rather skip it

Reply:
I was thinking we'd definitely want it to be multi-arch so that there's an ARM image as well. What's involved in that?

Reply:
Supporting `amd64` and `arm64`, as it does now, seems like it should cover most container hosts. It's perfect for my MacBook + Docker Desktop for example.

Reply:
Oh, maybe I misunderstood that arm64 was already supported. Agreed that those two should be sufficient for most cases. I'm going to approve and allow this to run through CI. @smira can `/lgtm` it if he's good with everything in the AM.